settlement/bitflyer/upload.go

```diff
@@ -386,41 +382,36 @@ func IterateRequest(
 		)
 	}
 
-	quote, err := bitflyerClient.FetchQuote(ctx, "BAT_JPY", true)
-	if err != nil {
-		return nil, err
-	}
 	for i, batch := range transactionBatches {
 		for j, tx := range batch {
 			tx.ProviderID = tx.BitflyerTransferID()
 			batch[j] = tx
 		}
 		transactionBatches[i] = batch
-	}
 
-	requests, err := createBitflyerRequests(
-		sourceFrom,
-		dryRun,
-		quote.PriceToken,
-		transactionBatches,
-	)
-	if err != nil {
-		return nil, err
-	}
+		quote, err := bitflyerClient.FetchQuote(ctx, "BAT_JPY", true)
```

Comment:
let's add a comment here, that this will only fetch a new quote when needed - but ensures that we don't have problems due to quote expiring midway through

Reply:
done